src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.ts

```diff
@@ -270,6 +270,13 @@ export class TicketInfoComponent implements OnInit {
       })
   }
 
+  /**
+   * Returns true if this Dispute is editable.
+   * @returns 
+   */
+  public isTicketInformationReadOnly() : boolean {
+    return this.lastUpdatedDispute?.status === this.DispStatus.Validated;
```

Comment:
I suggest we can make this as `public isTicketInformationReadOnly: boolean = false;`
After getting the ticket information, `this.isTicketInformationReadOnly = this.lastUpdatedDispute?.status === this.DispStatus.Validated;`
because it should be static

Reply:
Done. Moved to member variable.